.github/workflows/ci.yml

```diff
@@ -282,8 +280,10 @@ jobs:
       fail-fast: true
       matrix:
         build:
-          - { os: macos-11, xcode: 13.2.1, deployment: 11.3 }
-          - { os: macos-12, xcode: 14.2,   deployment: 12.5 }
+          - { os: macos-12,    xcode: 14.2,   deployment: 12.5 } # newest
+          - { os: macos-12,    xcode: 14.2,   deployment: 11.3 } # newest, backdeploy to oldest
```

Comment:
```suggestion
```

compilation is possible but leads to tons of linker warnings because the Homebrew dependencies are built for macos-12